resources/sdk/purecloudjava/templates/README.mustache

```diff
@@ -239,8 +239,8 @@ NotificationHandler notificationHandler = NotificationHandler.Builder.standard()
         .withNotificationListener(new UserPresenceListener(me.getId()))
         .withNotificationListener(new ChannelMetadataListener())
         // As a list
-        .withNotificationListeners(new ArrayList<NotificationListener<?>>()\{\{
-            add(new UserPresenceListener(me.getId());
+        .withNotificationListeners(new ArrayList<NotificationListener<?>>(){{
```

Comment:
I think the backslash was there because double curlys are reserved syntax in developer center templating. Can a space be added in between the two opening curlys? (here and the one on line 259)